Resources/Prototypes/Entities/Structures/Wallmounts/fireaxe_cabinet.yml

```diff
@@ -3,9 +3,21 @@
   name: fire axe cabinet
   description: There is a small label that reads "For Emergency use only" along with details for safe use of the axe. As if.
   components:
-  - type: Damageable # adding destructible causes the entity inside to be deleted when the cabinet is destroyed :(
```

Comment:
Does the axe now appear when cabinet is destroyed?

Reply:
From what I've seen, yes. I posted screenshots.